index.js

```diff
@@ -52,79 +52,79 @@ const root =
   // clicked on (in terminals such as iTerm) to open the file in an editor.
   `${process.cwd()}/`
 
-expandThunks = (array) => array.map(
-  elem => typeof elem == 'function' ? elem() : elem
-)
+const PRETTY = Boolean(process.env.PRETTY)
 
-makeLazyLogger = (strictLogger) => {
-  return strictLogger.enabled ?
-    // if logger enabled, call it with functions expanded to actual values
-    (...args) => strictLogger.apply(strictLogger, expandThunks(args)) :
-    // no-op
-    () => {}
-}
+const DEBUG = process.env.DEBUG
```

Comment:
after these changes, the lazy loggers don't do what they were designed to do: `expandThunks` should have only been called if logger was enabled